src/js/view/controls/controls.js

```diff
@@ -348,6 +374,14 @@ export default class Controls {
 
         // Hide controls when focus leaves the player
         const blurCallback = (evt) => {
+            //  Remove new shortcut tooltip after first read.
+            if (!OS.mobile) {
```

Comment:
This callback runs every time focus leaves the player, yet the logic you are adding should be called only once.
 I think we should have a separate callback which would be removed. Also, let's look into an alternative for `aria-describedby`, there might be a better tool. I think with `aria-live` we can alter the frequency with which something gets announced.

Reply:
As per our earlier discussion today, I'll leave this in for now until there's a better way to do it.